docs/using-the-compiler.rst

```diff
@@ -199,6 +199,13 @@ Output Description
             // The Ethereum Contract ABI. If empty, it is represented as an empty array.
             // See https://github.com/ethereum/wiki/wiki/Ethereum-Contract-ABI
             abi: [],
+            // See the Metadata Output documentation (serialised JSON string)
+            metadata: "{...}",
```

Comment:
We need to figure out what to do with the metadata. Maybe it makes sense to have:
- `metadataString` (serialised)
- `metadata` (object)

We'll know better when doing verification.